### PR TITLE
Translate the page "Licenses" (zh_cn)

### PR DESCRIPTION
**Pull request by shouxian**
Related #1074

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@shouxian :+1: Nice!

@pmq20 @hlcfan @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi @shouxian @johnnyhappy365

Please review this pull request, thank you!

Could you guys look at http://zh.wikipedia.org/zh-cn/BSD%E8%AE%B8%E5%8F%AF%E8%AF%81 (or find any other existing license translations) use the terms that is already known? E.g. 2-clause could be translated.

Thanks all!

File: zh_cn/about/license.txt [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Ruby 是受版权保护的自由软件，著作权人为 Yukihiro Matsumoto <matz@netlab.jp>.
+根据2-clause BSDL协议 (请参阅 BSDL 文件)和下述条款，你将被授权对它进行重新分发和（或）修改:
+
+  1. 你可以制作和（或）分发源代码格式的完整副本而不受限制，但必须包含原有的一切版权声明及免责声明。
+
+  2. 在满足下述条文中至少一项的前提下，你可以以任何方式修改你的副本:
+
+       a) 将您的修改部分发布于公共领域（或以其他形式确保公众能够免费得到修改后的副本），
+       包括（但不限于）在新闻组（或其他类似媒介）中张贴修改部分，或授权Ruby作者引用修改部分。
```
Comment (JuanitoFatas) [inline zh_cn/about/license.txt]: Also in my opinion, Ruby should add a space before & after:

```
或授权 Ruby 作者引用修改部分。
```

reads slightly better.

File: zh_cn/about/license.txt [hunk 2]
```diff
@@ -0,0 +1,33 @@
+Ruby 是受版权保护的自由软件，著作权人为 Yukihiro Matsumoto <matz@netlab.jp>.
```
Comment (VincentZhao) [inline zh_cn/about/license.txt]: Chinese peroid may be better.
「.」 → 「。」

File: zh_cn/about/license.txt [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+根据2-clause BSDL协议 (请参阅 BSDL 文件)和下述条款，你将被授权对它进行重新分发和（或）修改:
+
+  1. 你可以制作和（或）分发源代码格式的完整副本而不受限制，但必须包含原有的一切版权声明及免责声明。
+
+  2. 在满足下述条文中至少一项的前提下，你可以以任何方式修改你的副本:
+
+       a) 将您的修改部分发布于公共领域（或以其他形式确保公众能够免费得到修改后的副本），
```
Comment (VincentZhao) [inline zh_cn/about/license.txt]: For consistency.
「您」→「你」

File: zh_cn/about/license.txt (hunk 2 above)
Comment (JuanitoFatas) [inline zh_cn/about/license.txt]: :+1:

File: zh_cn/about/license.txt (hunk 3 above)
Comment (JuanitoFatas) [inline zh_cn/about/license.txt]: :+1:

File: zh_cn/about/license.txt [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+       包括（但不限于）在新闻组（或其他类似媒介）中张贴修改部分，或授权Ruby作者引用修改部分。
+
+       b) 只在你的公司或组织内部使用修改后的副本。
```
Comment (johnnyhappy365) [inline zh_cn/about/license.txt]: I think it should be better with "只在你所在的公司或组织" instead of "只在你的公司或组织"

File: zh_cn/about/license.txt [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+       d) 联系作者以取得其他形式的授权。
+
+  3. 在满足下述条文中至少一项的前提下，你可以以任何方式分发软件的源码或其二进制形式:
+
+       a) 在分发库文件或二进制文件时，提供关于可以从何处获取原始拷贝的说明。
+
+       b) 在分发中提供（机器可读的）源代码。
+
+       c) 当使用使用单独的产品名称以二进制形式分发的（已修改过的）拷贝时，同时提供可以从何处获取原始（未修改过的）拷贝的说明。
```
Comment (johnnyhappy365) [inline zh_cn/about/license.txt]: May be a duplicate of "使用"

File: zh_cn/about/license.txt [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+       d) 联系作者以取得其他形式的授权。
+
+  4. 你可以修改或将其作为其他软件（可以是商业的）的一部分。但对于并非由著作权人撰写的部分，不适用于本条款。
+
+     关于拷贝具体文件所需要的先决条件说明，请参阅 LEGAL 文件
+
+  5. 使用软件生成的脚本或库文件并不自动适用于本版权声明，它们的版权仍应属于生成者。
```
Comment (johnnyhappy365) [inline zh_cn/about/license.txt]: I think there is part of words lost rather than original list.

**Comment by JuanitoFatas** [comment]
@shouxian @johnnyhappy365 @VincentZhao My apologies guys. License does not need to be translated. Only english version is official.

**Pull request closed without merging**